3rdparty/p256-m/p256-m_driver_entrypoints.c

```diff
@@ -149,21 +229,23 @@ psa_status_t p256_transparent_sign_hash(
     (void) attributes;
     (void) alg;
 
-    psa_status_t status = PSA_ERROR_NOT_SUPPORTED;
-    if (key_buffer_size != 32 || signature_size != 64) {
-        return status;
+    /* Validate sizes, as p256-m expects fixed-size buffers */
+    if (key_buffer_size != PRIVKEY_SIZE) {
+        return PSA_ERROR_CORRUPTION_DETECTED;
+    }
+    if (signature_size < SIGNATURE_SIZE) {
+        return PSA_ERROR_BUFFER_TOO_SMALL;
```

Comment:
Documentation for this function does not contain all error codes:

`PSA_ERROR_BUFFER_TOO_SMALL` and `PSA_ERROR_CORRUPTION_DETECTED` are missing (along with anything that `p256_ecdsa_sign()` can return), `PSA_ERROR_NOT_SUPPORTED` needs removing.